conf/messages.en

```diff
@@ -891,38 +922,59 @@ declaration.summary.transaction.natureOfTransaction.7 = Military
 declaration.summary.transaction.natureOfTransaction.8 = Construction
 declaration.summary.transaction.natureOfTransaction.9 = Other
 declaration.summary.transaction.previousDocuments = Documents related to this declaration
+declaration.summary.transaction.previousDocuments.change = Change documents related to this declaration
 declaration.summary.transaction.previousDocuments.type = Document type
 declaration.summary.transaction.previousDocuments.reference = Document reference
+declaration.summary.transaction.previousDocuments.document.change = Change document {0}
 declaration.summary.items = About the items
 declaration.summary.items.empty = You have not added any items yet
 declaration.summary.items.item.sequenceId = Declaration item {0}
 declaration.summary.items.item.procedureCode = Procedure code
+declaration.summary.items.item.procedureCode.change = Change procedure code from item {0}
```

Comment:
I would personally have "Change x for item y" (rather than 'from') but I guess that wasn't your call.

Reply:
There wasn't anything about it in the ticket. I asked designers about it during implementation and they approved it. I think it requires more discussion. Let's leave it for now

Reply:
If it wasn't on the ticket then I'd either ask design or do a quick replace text.

Reply:
Yup, you're right, confirmed and changing